common/helpers/dateHelperUTCFormatted.js

```diff
@@ -37,6 +37,12 @@ function timeDurationShort(start, end, tc) {
   })
 }
 
+// short format of dateTime range
```

Comment:
Here you could use jsdoc, this would appears in the ide when you hover over the function or in the autocompletion when you use the function

Reply:
This code is part of #3411